assets/src/edit-story/components/panels/alignment/alignment.js

```diff
@@ -54,6 +54,8 @@ const ElementRow = styled.div`
   align-items: center;
   padding: 10px 20px;
   border-top: 1px solid ${({ theme }) => theme.colors.bg.v9};
+  overflow-y: overlay;
+  overflow-x: overlay;
```

Comment:
Is there a reason why just not to use `overflow`? Also, isn't `overlay` deprecated / not supported by FF?